rotkehlchen/tests/integration/test_price_history.py

```diff
@@ -46,6 +46,6 @@ def test_price_queries(price_historian, database):
     price_historian.set_oracles_order(price_historian._oracles)
     assert price_historian.query_historical_price(A_DASH, A_USD, 1438387700) == FVal('10')
     # this should hit coingecko, since cornichon is not in cryptocompare
-    expected_price = FVal('0.5771301464712261')
-    yvecrv = EvmToken('eip155:1/erc20:0xc5bDdf9843308380375a611c18B50Fb9341f502A')
-    assert price_historian.query_historical_price(yvecrv, A_USD, 1704135600) == expected_price
+    expected_price = FVal('1.00036501912699')
+    yusdt = EvmToken('eip155:1/erc20:0x2f08119C6f07c006695E079AAFc638b8789FAf18')
+    assert price_historian.query_historical_price(yusdt, A_USD, 1704135600) == expected_price
```

Comment:
`yvecrv` historical price was moved to coinbase pro api.

Reply:
what do you mean by this?